draft-ietf-suit-report.md

```diff
@@ -627,48 +618,56 @@ Label | Name | Reference
-security considerations. In situations where the SUIT\_Report is
-encrypted as an element of a message within another protocol, care must
-be taken to ensure that this does not leak information and that the
-principle of least privilege is respected. For example, in an EAT-based
-attestation workflow, the Verifier often will not need the full SUIT\_Report.
-Similarly, the Relying Party may also not need the SUIT\_Report.
-To enable the principle of least privilege in this and similar
-scenarios, the SUIT\_Report should be independently encrypted even if
-the EAT token or encrypted transport that contains it is also encrypted.
-
-In contrast, however, there are scenarios where the EAT Verifier
-consumes the SUIT\_Report and translates it into one or more other
-EAT claims. For example, a SUIT\_Report that shows a particular digest
-was matched using an suit-condition-image can be translated into a
-EAT measres (Measurement Results) claim. In this scenario, the Verifier
-must have access to the full SUIT\_Report.
+#  Security Considerations {#seccons
+
+The SUIT\_Report serves four primary security objectives:
+
+- Authenticity  
+- Integrity  
+- Replay protection  
+- Confidentiality
+
+The mechanisms for achieving these protections are outlined in {{container}}.
+
+Ideally, a SUIT\_Report SHOULD be conveyed as part of an attestation process,
+such as embedded within an EAT token. This approach ensures that the SUIT\_Report
+is cryptographically bound to the environment (hardware, software, or both) in
+which it was generated, thereby strengthening its authenticity.
+
+A SUIT\_Report may disclose sensitive information about the device on which they
+were produced. In such cases, the SUIT\_Report MUST be encrypted, as specified in
+{{container}}.
+
+Furthermore, failure reports, particularly those involving cryptographic operations,
+can unintentionally reveal insights into system weaknesses or vulnerabilities. As such,
+SUIT\_Reports SHOULD be encrypted whenever possible, to minimize the risk of information
+leakage.
+
+In addition to these core security requirements, operational considerations must be taken
+into account. When a SUIT\_Report is included within another protocol message (e.g., inside
+an encrypted EAT token), care must be taken to avoid inadvertently leaking information and
+to uphold the principle of least privilege. For example, in many EAT-based attestation flows,
+the Verifier may not require the full SUIT\_Report. Similarly, the Relying Party might not
+need access to it either.
+
+To support least-privilege access, the SUIT\_Report should be independently encrypted, even
+when the transport or enclosing token is also encrypted. This layered encryption ensures that
+only authorized entities can access the contents of the SUIT\_Report.
+
+On the other hand, in scenarios where the EAT Verifier requires full access to the SUIT\_Report.
+For example, to extract or convert its content into specific EAT claims, such as `measres`
```

Comment:
What is "its"? The message? The Verifier? I cannot parse this sentence.

Reply:
I believe you changed the sentence already to make it clear.